src/plug/Widgets/UserItem.vala

```diff
@@ -87,7 +87,8 @@ namespace PC.Widgets {
             master_switch.sensitive = Utils.get_permission ().get_allowed ();
 
             full_name_label.label = user.get_real_name ();
-            username_label.label = "<span font_size=\"small\">%s</span>".printf (GLib.Markup.escape_text (user.get_user_name ()));
+            var escaped_name = GLib.Markup.escape_text (user.get_user_name ());
+            username_label.label = "<span font_size=\"small\">%s</span>".printf (escaped_name);
```

Comment:
While you're at it :slightly_smiling_face: 
```suggestion
            username_label.label = GLib.Markup.printf_escaped ("<span font_size=\"small\">%s</span>", user.get_user_name ())
```

Reply:
That's better - but slightly overlong as it is.  I was trying not to change the code as far as possible. I'll make an exception here though.